indico/modules/events/papers/operations.py

```diff
@@ -302,6 +321,15 @@ def create_comment(paper, text, visibility, user):
     comment = PaperReviewComment(user=user, text=text, visibility=visibility)
     paper.last_revision.comments.append(comment)
     db.session.flush()
+    receivers = {x for x in paper.cfp.judges}
```

Comment:
Why email judges not assigned to the paper? The might not even be able to see that paper.